<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>色谱</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>791</width>
     <height>351</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tab_auto">
    <attribute name="title">
     <string>正常模式</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>731</width>
       <height>290</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pb_solution_setting">
          <property name="text">
           <string>方法配置...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pb_send">
          <property name="text">
           <string>方法传送   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pb_ready">
          <property name="text">
           <string>手动准备</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Warmming">
          <property name="text">
           <string>预热</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_run">
          <property name="text">
           <string>运行</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_3"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pb_stop">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_factory">
    <attribute name="title">
     <string>工厂模式</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>230</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>190</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>230</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>171</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_action_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_action_4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>190</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_action_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_action_1">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>41</y>
       <width>86</width>
       <height>244</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="sw_1">
        <property name="text">
         <string>电磁阀1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sw_2">
        <property name="text">
         <string>电磁阀2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sw_3">
        <property name="text">
         <string>电磁阀3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sw_4">
        <property name="text">
         <string>电磁阀4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sw_5">
        <property name="text">
         <string>电磁阀5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sw_6">
        <property name="text">
         <string>电磁阀6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sw_7">
        <property name="text">
         <string>电磁阀7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sw_8">
        <property name="text">
         <string>电磁阀8</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar_1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDial" name="dial_pad">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>70</y>
       <width>111</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>90</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="sliderPosition">
      <number>42</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchTarget">
      <double>9.699999999999999</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>220</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>后开门</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>170</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>32</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_dump">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>dump</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>运行曲线</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>日志维护</string>
    </attribute>
    <widget class="QPushButton" name="pb_export">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>导出...</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>496</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>开始时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>结束时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>181</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>串口数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_10">
          <property name="text">
           <string>以太网数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_11">
          <property name="text">
           <string>日志数据 </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>801</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_state">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>20</number>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_info">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>20</number>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>后开门</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar_Door">
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="format">
       <string>%p°</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="pb_halt">
      <property name="text">
       <string>紧急复位</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_autotest">
      <property name="text">
       <string>自检信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_auto">
      <property name="text">
       <string>正常模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_factory">
      <property name="text">
       <string>工厂模式</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBoxSw1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSw1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电磁阀1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSw2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSw2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电磁阀2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSw3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSw3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电磁阀3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSw4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSw4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电磁阀4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSw5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSw6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电磁阀6</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSw7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电磁阀7</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSw8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电磁阀8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSw7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSw8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSw6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSw5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电磁阀5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>辅温箱1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ex0tmp">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>96</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>04b_21</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>辅温箱2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ex1tmp">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>96</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>辅温箱3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ex2tmp">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>96</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>FID温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fid">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>125</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TCD温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tcd">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>125</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_main">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>125</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>主控温度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
